src/opt_methods/IPNewton.jl

```diff
@@ -6,33 +6,33 @@ mutable struct IPNewton{T <: Real} <: AbstractOptimizationMethod{T}
     ub::Vector{T}
     lb::Vector{T}
     iterations::Int
-    seeds::Vector{Vector{T}}
+    initial_points::Vector{Vector{T}}
 end
 
 """
     IPNewton(model::Model, args...; kwargs...)
 
-If an `Int` is provided the method will sample random seeds and use them as initial points for Optim IPNewton method.
+If an `Int` is provided the method will sample random initial_points and use them as initial points for Optim IPNewton method.
 If a `Vector{Vector{T}}` is provided it will use them as initial points for Optim IPNewton method.
 """
-function IPNewton(model::Model{D, T}, n_seeds::Int; f_tol::T = T(1e-6), g_tol::T = T(1e-6), 
+function IPNewton(model::Model{D, T}, n_initial_points::Int; f_tol::T = T(1e-6), g_tol::T = T(1e-6), 
                                                 ub::Vector{T} = Inf*ones(T, dim_unknowns(model)),
                                                 lb::Vector{T} = -Inf*ones(T, dim_unknowns(model)),
                                                 iterations::Int = 10^5, LB::T = 0.0, UB::T = 0.6) where {D, T}
 
-    seeds = create_seeds(model, n_seeds, LB, UB)
+    initial_points = create_seeds(model, n_initial_points, LB, UB)
```

Comment:
Should probably change the name of this function as well

Reply:
for sure! Thanks